### ISSUE TITLE
Always add a background color for PNG > JPEG

### ISSUE THREAD

**Issue by jacobsvante** [jazzband/sorl-thumbnail#96]
Converting a transparent PNG to JPEG with sorl should reasonably create an image with simple background color put "underneath" the image (reasonably defaulting to white as it's the most common background color on the web). However I get results similar to what is described in [this post](http://stackoverflow.com/questions/9166400/convert-rgba-png-to-rgb-with-pil) both with pgmagick and PIL set as engines.

So simply put: sorl should detect if the original image has an alpha channel that is to be converted to a non-alpha format, and if true, put a simple color layer underneath the image before creating the thumbnail.
